Cargo.toml

```diff
@@ -52,12 +52,14 @@ serde = { version = "1.0", default-features = false, optional = true, features =
 packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_bits"], optional = true }
 zeroize = { version = "1", default-features = false }
 fiat-crypto = { version = "0.1.6", optional = true}
+group = { version = "0.11", default-features = false, optional = true }
 
 [features]
 nightly = ["subtle/nightly"]
 default = ["std", "u64_backend"]
 std = ["alloc", "subtle/std", "rand_core/std"]
 alloc = ["zeroize/alloc"]
+elliptic-curve = ["group"]
```

Comment:
Some people don't need it, that's the only reason, saves compile time and makes it easier to cut down on dependencies that need to be relied on and audited.

That's the way I'm used to build libraries, but I'm happy to remove it. Another motivation would be to keep the MSRV for the rest of the project and just mark the crate feature alone to have a higher MSRV.

Reply:
It would probably make sense not to add extra indirection here yet. You can use `group` as a feature since the crate is `optional = true`, without adding a superfluous feature.

Reply:
The plan was to continue adding more traits from elliptic-curve as a follow-up, but adding the necessary crate feature can be done in the follow-up PR too.

Reply:
If you gate the group-related traits on `group`, they can be used independently of the `elliptic-curve` crate, which will pull in more dependencies.

It seems like you should only need to pull in `elliptic-curve` for things like hash2curve.

Reply:
That's a good idea, thank you, I will do that then!